src/mixins/java/org/spongepowered/common/mixin/inventory/impl/world/inventory/ItemCombinerMenuMixin_TrackedMenuBridge_Inventory.java

```diff
@@ -36,6 +39,8 @@
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 import org.spongepowered.common.bridge.world.inventory.container.TrackedMenuBridge;
 
+import javax.annotation.Nullable;
```

Comment:
```suggestion
import org.checkerframework.checker.nullness.qual.Nullable;
```

We don't use the javax annotations.